libs/common-system/pom.xml

```diff
@@ -38,6 +38,11 @@
             <artifactId>slf4j-api</artifactId>
             <version>${org.slf4j.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.servicemix.bundles</groupId>
+            <artifactId>org.apache.servicemix.bundles.jsr305</artifactId>
+            <scope>provided</scope>
```

Comment:
✏️ It's fine to include the scope here, but as this artifact is dependencyManaged, it would only really matter if you were overriding the scope; just as you inherit the version, you inherit the scope.